setup_utils/find_libfuzzer.sh

```diff
@@ -24,11 +24,11 @@ if [[ "$uname" == "Darwin" ]]; then
 elif [[ "$uname" == "Linux" ]]; then
   machine="$(uname -m)"
   if [[ "$machine" == "x86_64" ]]; then
```

Comment:
Note: I haven't fixed OSX because I have no way to test this or determine the triple correctly.